script/test262.js

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env node
```

Comment:
Before we can accept this PR, this source file should include a description of:
- what it's intended for (including when should it run, who should run it),
- what it does,
- how it's achieving it,
- how it should be maintained and evolve (including how to manage the file(s) listing expected failures)

Reply:
I think the most important is to ensure:
* a PR is blocked when there's a regression or a progression;
* print what to do in case of a regression or unexpected failure: how to locate the source snippet, how to mark it as an expected failure if needed;
* print what to do if a test that used to fail now passes: how to remove it from the list of expected failures